docs/RELEASE-techpreview.md

```diff
@@ -0,0 +1,29 @@
+OSP Director Operator Tech Preview
+==================================
+
+Overview
+--------
+The OSP Director Operator is an OpenShift Container Platform (OCP) “Operator” that streamlines the installation of OpenStack on OpenShift. The OSP Director Operator creates a set of Custom Resource Definitions (CRDs) on top of OpenShift to manage resources normally created by the TripleO Undercloud. These CRDs are split into two types for hardware provisioning and software configuration which aligns well with TripleO's Undercloud Lite architecture.
+
+These CRDs allow administrators to provision and scale out virtual and baremetal hardware for an OSP deployment. Virtual machines for the OSP control plane are provisioned via Kubevirt(OpenShift Virtualization). Baremetal for OSP compute hosts is provisioned via Metal3. And OvercloudNet provides a simple IPAM solution for IPv4 and IPv6 static IP assignments.
+
+Supported Features
+------------------
+
+The following table highlights features supported today in OSP 16.2 compared with future planned feature work for OSP 17 (GA).
```

Comment:
Should we mention something about HA in the features?

Reply:
You mean like HA would require a fencing agent?

Reply:
Sure.  Just something to indicate that "full HA" has not yet been addressed.

Reply:
just pushed an update to include this. thanks!